system/tests/test_logmessaged.py

```diff
@@ -46,6 +47,7 @@ def test_simple_log(self, temp_dir):
     assert len(self._get_log_files()) >= 1
 
   @temporary_swaglog_dir
+  @temporary_swaglog_ipc
```

Comment:
this common stuff should be done more generically. can we run all tests by default in an OpenpilotPrefix?

Reply:
you mean a temporary directory that everything uses? maybe we should have a param for "TempWorkspace" and everything uses that as a base?

Reply:
we already have this (https://github.com/commaai/openpilot/blob/9719b227611c880c76a60df223ef6c2f14f1a6ab/selfdrive/test/process_replay/helpers.py#L9), but can we use it by default?

Reply:
I think that will require a bunch of refactoring, okay if I move that to a separate PR? I'll make an issue for it